src/less/variables/ds6/border-radius-variables.less

```diff
@@ -12,7 +12,7 @@
 @border-radius-dropdown: @border-radius-form-control;
 @border-radius-dropdown-box-shadow: 0 5px 15px rgba(0, 0, 0, 0.07);
 @border-radius-dropdown-border-color: @color-grey1;
-@border-radius-notice: @border-radius-small;
+@border-radius-notice: @border-radius-none;
```

Comment:
If we are no longer supporting a border-radius on notice, we can remove this variable and the `border-radius` on notice.